src/main/io/osd.c

```diff
@@ -479,6 +480,42 @@ static void osdDrawSingleElement(uint8_t item)
         break;
     }
 
+    case OSD_MAIN_BATT_USAGE:
+    {
+        //Set length of indicator bar
+        uint8_t progressSteps = 10;
+
+        //Calculate constrained value
+        float value = constrain(osdConfig()->cap_alarm - getMAhDrawn(), 0, osdConfig()->cap_alarm);
```

Comment:
It's probably better to use `batteryConfig()->batteryCapacity` here. (I am wondering if `osdConfig()->cap_alarm` should be removed in favour of `batteryConfig()->cbat_alert_percent` anyway. Not much point in having battery related settings in `osdConfig()`.